### DISCUSSION TITLE
Best way to add screenshots to a post

### DISCUSSION THREAD

**Discussion by casadesk** [GetPublii/Publii#1410] [Help]
I’m looking for the best way to include screenshots and other images into a post.

Easiest way would be if I was able to paste an image from the clipboard, but that does not seem to be possible (I found several requests for this feature but it doesn’t seem to be on the roadmap; is there a roadmap?).

Did anyone found an easier way to save every screenshot locally and then upload it into Publii?

**Comment by example-user30** [comment]
Save the screen shot or image as a file.   Choose insert/edit image from the menu in the block editor or WYSIWYG editor.  Drag it from the filesystem into the text area for Markdown editor.  This is covered in the docs for block editor: [https://getpublii.com/docs/the-block-editor.html](https://getpublii.com/docs/the-block-editor.html) and for WYSIWYG editor: [https://getpublii.com/docs/post-editor.html](https://getpublii.com/docs/post-editor.html)

**Reply by casadesk** [reply]
Clear. Thanks!
